### ISSUE TITLE
[Feature Request] Intercom hot mic support for F-15e

### ISSUE THREAD

**Issue by burnzoire** [ciribob/DCS-SimpleRadioStandalone#663]
According to the F-15E manual, the intercom has 3 settings based on the MIC 3-way switch in the left console:
RAD ORIDE: attenuate radio comms in favour of intercom communication. Warning messages are not overridden.
ON: Direct communication from the cockpit in which it is selected with the other cockpit and/or ground crew (i.e front/rear both set this on and it should continuously transmit)
OFF: turns of mic for intercom purposes but can still operate radios.

TL;DR: There is no switch for transmitting over the intercom, it's either always transmitting or not. Can we please get support for this in the streagle?
